dropwizard-demo/src/main/java/com/dep/Services/TwitterImplement.java

```
@@ -49,25 +54,50 @@ public Status sendTweets(String tweet) throws TwitterException {
-                name = status.getUser().getName();
-                profileImageUrl = status.getUser().getProfileImageURL();
+                String message = status.getText();
+                String name = status.getUser().getName();
+                String profileImageUrl = status.getUser().getProfileImageURL();
                 createdAt = status.getCreatedAt();
                 Format format = new SimpleDateFormat("dd-mm-yyy HH:mm:ss");
                 String date = format.format(createdAt);
-                twitterHandle = status.getUser().getScreenName();
+                String twitterHandle = status.getUser().getScreenName();
                 tweetRespons = new TweetRespons(message, name, twitterHandle, profileImageUrl, date);
                 arrayList.add(tweetRespons);
             }
         } catch (TwitterException e) {
         }
         return arrayList;
     }
+
+
+    public List<TweetRespons> getFilteredTweets(String tweets) {
+        ArrayList<TweetRespons> arrayList = new ArrayList<>();
+        List<TweetRespons> filteredTweets;
+        List<Status> statuses = null;
+        try {
+            statuses = twitter.getHomeTimeline();
+            for (int i = 0; i < statuses.size(); i++) {
+                Date createdAt ;
+                Status status = statuses.get(i);
+                String profileImageUrl = status.getUser().getProfileImageURL();
+                String name = status.getUser().getName();
+                String message = status.getText();
+                createdAt =status.getCreatedAt();
+                Format format = new SimpleDateFormat("dd-mm-yyy HH:mm:ss");
+                String date = format.format(createdAt);
+                String twitterHandle = status.getUser().getScreenName();
+                tweetRespons = new TweetRespons(message,name,twitterHandle,profileImageUrl,date);
+                arrayList.add(tweetRespons);
+            }
+        } catch (TwitterException e) {
+        }
```

Apply DRY , Empty catch block ?  add a log